fix: revise active learning exercise

CBID2 opened the pull request:
<!-- 🙌 Thanks for contributing to MDN Web Docs. Adding the details below will help us to merge your PR faster. -->

### Description

<!-- ✍️ Summarize your changes in one or two sentences -->
I omitted the last bullet point the Active Learning exercise and revised the exercise's title. 
### Motivation

<!-- ❓ Why are you making these changes and how do they help readers? -->
Someone pointed it out that adding articles in HTML was not discussed in the earlier parts of the article. Making these edits would make it easier for readers to understand how to do the exercise. 

### Additional details

<!-- 🔗 Link to release notes, vendor docs, bug trackers, source control, or other places providing more context -->
Here's [the lesson I edited](https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Getting_started).

### Related issues and pull requests

<!-- 🔨 If this fully resolves a GitHub issue, use "Fixes #123" -->
<!-- 👉 Highlight related pull requests using "Relates to #123" -->
<!-- ❗ If another pull request should be merged first, use "**Depends on:** #123" -->
Fixes #27649 

<!-- 👷‍♀️ After submitting, go to the "Checks" tab of your PR for the build status -->

CBID2 committed:
fix: add aria-label to the form example

hamishwillee committed:
Fix up errors by running prettier in vscode

hamishwillee committed:
Apply suggestions from code review

CBID2 committed:
Merge branch 'main' into main

CBID2 committed:
Merge branch 'mdn:main' into main

CBID2 committed:
Merge branch 'mdn:main' into main

CBID2 committed:
fix: omit last instruction in the Active learning exercise 

Someone pointed out that the last bullet point in this exercise made it difficult for them to understand since adding images is not discussed in the earlier portion of the article, so I removed it to make the directions clearer.

CBID2 committed:
fix: revise formatting

CBID2 committed:
Merge branch 'main' into CBID2-revise-active-learning-exercise

CBID2 committed:
revise formatting

CBID2 committed:
Merge branch 'main' into CBID2-revise-active-learning-exercise

CBID2 committed:
Merge branch 'main' into CBID2-revise-active-learning-exercise

CBID2 committed:
Merge branch 'main' into CBID2-revise-active-learning-exercise

dipikabh commented on files/en-us/learn/html/introduction_to_html/getting_started/index.md:
```diff
@@ -514,15 +514,14 @@ If you want to experiment with writing some HTML on your local computer, you can
 - Edit the paragraph content to include text about a topic that you find interesting.
 - Make important words stand out in bold by wrapping them inside a `<strong>` opening tag and `</strong>` closing tag.
 - Add a link to your paragraph, as [explained earlier in the article](#active_learning_adding_attributes_to_an_element).
-- Add an image to your document. Place it below the paragraph, as [explained earlier in the article](#empty_elements). Earn bonus points if you manage to link to a different image (either locally on your computer, or somewhere else on the web).
```
I believe this point does not need not be removed. Only the link needs to be updated (I guess the section name changed at some point and led to this broken link). Using the `<img>` element is briefly discussed in this section https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Getting_started#void_elements.

CBID2 replied:
Oh thanks for pointing this out @dipikabh.

dipikabh replied:
Perhaps in your next commit for this PR fix, you can reinstate that bullet point along with the updated link

CBID2 replied:
> Perhaps in your next commit for this PR fix, you can reinstate that bullet point along with the updated link

Made the change @dipikabh

dipikabh commented on files/en-us/learn/forms/how_to_structure_a_web_form/index.md:
```diff
@@ -52,15 +52,15 @@ Here is a little example:
     <legend>Fruit juice size</legend>
     <p>
-      <input type="radio" name="size" id="size_1" value="small" />
+      <input type="radio" name="size" id="size_1" value="small" required />
```
Sorry, I am missing the context for the changes in this file

dipikabh replied:
Could you explain the reason/motivation to update this file? I do not see a reference to this change in your PR description. Was this raised as part of the original issue?

CBID2 replied:
Hi @dipikabh. This particular code line was a part of [this issue, which has been solved.](https://github.com/mdn/content/issues/26790), so you can ignore it.

dipikabh replied:
Actually, I cannot ignore these changes because I will be the one approving all the changes in this pull request :).
Could I request you to revert the changes to this file (please open a new PR for these changes) and let this PR be only about the issue you're fixing? It will help reviewers in future if you address one issue per PR. Thanks!

dipikabh reviewed (changes requested):
Hello @CBID2, thanks for opening this PR to fix the issue!

I've added some suggestions and updates for the changes.

CBID2 committed:
Update files/en-us/learn/html/introduction_to_html/getting_started/index.md

Co-authored-by: Dipika Bhattacharya <dipika@foss-community.org>

CBID2 committed:
Update files/en-us/learn/html/introduction_to_html/getting_started/index.md

Co-authored-by: Dipika Bhattacharya <dipika@foss-community.org>

CBID2 committed:
Update files/en-us/learn/html/introduction_to_html/getting_started/index.md

Co-authored-by: Dipika Bhattacharya <dipika@foss-community.org>

CBID2 committed:
Update files/en-us/learn/html/introduction_to_html/getting_started/index.md

Co-authored-by: Dipika Bhattacharya <dipika@foss-community.org>

CBID2 committed:
Update files/en-us/learn/html/introduction_to_html/getting_started/index.md

Co-authored-by: Dipika Bhattacharya <dipika@foss-community.org>

dipikabh commented on files/en-us/learn/html/introduction_to_html/getting_started/index.md:
```diff
@@ -748,10 +747,11 @@ You made it to the end of the article! We hope you enjoyed your tour of the basi
 - [Applying color to HTML elements using CSS](/en-US/docs/Web/CSS/CSS_Colors/Applying_color)
 
 {{NextMenu("Learn/HTML/Introduction_to_HTML/The_head_metadata_in_HTML", "Learn/HTML/Introduction_to_HTML")}}
+
```
```suggestion
```
markdownlint reporting an extra line

dipikabh reviewed (changes requested):
Hello @CBID2, thanks for the fixes. Two comments need addressing. We're so close to landing this. :)

CBID2 commented:
> Hello @CBID2, thanks for the fixes. Two comments need addressing. We're so close to landing this. :)

Thanks @dipikabh. Still figuring out how to put the link about the image. Do you have any suggestions?

CBID2 committed:
fix: revise grammatical errors and added the third bullet point back

One of the maintainers presented a link that contained the information about images, so I placed the third bullet point again and updated the link.

CBID2 committed:
Update files/en-us/learn/html/introduction_to_html/getting_started/index.md

Co-authored-by: Dipika Bhattacharya <dipika@foss-community.org>

CBID2 committed:
Merge branch 'main' into CBID2-revise-active-learning-exercise

CBID2 commented:
> > Hello @CBID2, thanks for the fixes. Two comments need addressing. We're so close to landing this. :)
> 
> Thanks @dipikabh. Still figuring out how to put the link about the image. Do you have any suggestions?

I added the link you mentioned and I got this warning:
"Warning: files/en-us/learn/html/introduction_to_html/getting_started/index.md:525:100 search-replace Custom rule [fqdn-moz-links: Don't use developer.mozilla.org for links, except for blog posts] [Context: "column: 100 text:'](https://developer.mozilla.org/'"]"

dipikabh commented on files/en-us/learn/html/introduction_to_html/getting_started/index.md:
```diff
@@ -514,15 +514,15 @@ If you want to experiment with writing some HTML on your local computer, you can
 - Make important words stand out in bold by wrapping them inside a `<strong>` opening tag and `</strong>` closing tag.
 - Add a link to your paragraph, as [explained earlier in the article](#active_learning_adding_attributes_to_an_element).
-- Add an image to your document. Place it below the paragraph, as [explained earlier in the article](#empty_elements). Earn bonus points if you manage to link to a different image (either locally on your computer, or somewhere else on the web).
+- Add an image to your document. Place it below the paragraph, as [explained earlier in the article](https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Getting_started#void_elements). Earn bonus points if you manage to link to a different image (either locally on your computer or somewhere else on the web).
```
In the links on MDN, we use part of the slug starting from "/en-US". So the link below will work.

```suggestion
- Add an image to your document. Place it below the paragraph, as [explained earlier in the article](/en-US/docs/Learn/HTML/Introduction_to_HTML/Getting_started#void_elements). Earn bonus points if you manage to link to a different image (either locally on your computer or somewhere else on the web).
```

dipikabh reviewed (changes requested):
Hope this helps, @CBID2.

CBID2 committed:
fix:  apply dipikabh's suggestion

Co-authored-by: Dipika Bhattacharya <dipika@foss-community.org>

CBID2 committed:
Merge branch 'main' into CBID2-revise-active-learning-exercise

CBID2 commented:
> Hope this helps, @CBID2.

Made the change @dipikabh

CBID2 committed:
Merge branch 'main' into CBID2-revise-active-learning-exercise

dipikabh commented:
Thank you! Could you also revert changes to files/en-us/learn/forms/how_to_structure_a_web_form/index.md as mentioned here https://github.com/mdn/content/pull/27782#discussion_r1261813592.

CBID2 commented:
> Thank you! Could you also revert changes to files/en-us/learn/forms/how_to_structure_a_web_form/index.md as mentioned here [#27782 (comment)](https://github.com/mdn/content/pull/27782#discussion_r1261813592).

I can't revert the changes @dipikabh. Can you try on your end?

CBID2 commented:
Making a new PR. Hold on

CBID2 commented:
H

CBID2 commented:
Here's the new version of this PR @dipikabh: https://github.com/mdn/content/pull/27910
